app.js

```diff
@@ -66,6 +84,11 @@ class Application {
           ? JSON.parse(req.files.iterationDataFile.data.toString())
           : null;
 
+        logger.log(
+          LogLevel.info,
+          `Running Postman collection: ${collectionFileJSON.info.name}`
```

Comment:
can you "null protect" this, I don't want the server to crash is the collectionFile does not have any "info" prop

Reply:
Done